### PR TITLE
execution: add round function

### PR DESCRIPTION
**Pull request by MichaHoffmann**
Partially addresses #138

### CODE REVIEW

File: engine/engine_test.go [hunk 1]
```diff
@@ -1819,6 +1819,28 @@ func TestInstantQuery(t *testing.T) {
 			queryTime: time.Unix(160, 0),
 			query:     "increase(http_requests_total[1m] offset 1m)",
 		},
+		{
+			name: "round with argument",
```
Comment (saswatamcode) [inline engine/engine_test.go:1823]: I think the test case name should be switched here?
```suggestion
			name: "round without argument",
```

File: engine/engine_test.go [hunk 2]
```diff
@@ -1819,6 +1819,28 @@ func TestInstantQuery(t *testing.T) {
...
+				       http_requests_total{pod="nginx-1", series="1"} 1+1.1x40
+				       http_requests_total{pod="nginx-2", series="2"} 2+2.3x50
+				       http_requests_total{pod="nginx-4", series="3"} 5+2.4x50
+				       http_requests_total{pod="nginx-5", series="1"} 8.4+2.3x50
+				       http_requests_total{pod="nginx-6", series="2"} 2.3+2.3x50`,
+			queryTime: time.Unix(0, 0),
+			query:     "round(http_requests_total)",
+		},
+		{
+			name: "round without argument",
```
Comment (saswatamcode) [inline engine/engine_test.go:1834]:
```suggestion
			name: "round with argument",
```

**Review by saswatamcode — APPROVED** [review]
LGTM, mod some suggestions! 🙂

File: engine/engine_test.go (hunk 1 above)
Comment (MichaHoffmann) [reply engine/engine_test.go]: thanks, addressed it!

**Review by fpetkovski — APPROVED** [review]
Thanks

**Pull request merged**